### ISSUE TITLE
Inaccessible footer styles

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#2005]
## Problems
Customers are getting into trouble with their footer markup. Much of the filtered_html markup available has not been accounted for as well as common footer markup like `<address>`.

Until we have more guidance about what content should go into the footer area, we need to address the content that is already out there.

## Proposed Solution

Extend the same coverage as bg-black text paragraphs styles have locally in uids_base and address policy and the future of footer styles in uids.
